tests/Application.IntegrationTests/TestHelpers/DatabaseContainer.cs

```diff
@@ -0,0 +1,35 @@
+using Testcontainers.SqlEdge;
+
+namespace Application.IntegrationTests.TestHelpers;
+
+public class DatabaseContainer
+{
+    private readonly SqlEdgeContainer? _container;
+
+    public DatabaseContainer()
+    {
+        ConnectionString = Environment.GetEnvironmentVariable("ConnectionStrings__DefaultConnection");
```

Comment:
Is there a reason why you're pulling this from an environment variable instead of from a generic `IConfiguration`?

Reply:
Not really - Actually I'll remove this one since we are not consuming any appsettings or configuration at all

Reply:
It wasn't there as an configuration concern, hence no `IConfiguration`. It was there if you needed to inject the connection string from it being run in CI environment i.e. BitBucket blocks calls to the docker daemon, so we'd set up the sql server as part of the pipeline definition and inject the connection string into the test container as an override.

@GordonBeeming found some other flag that could be set in that scenario

Reply:
I think the configuration above should not be in the template since it is only used for one particular scenario - should be handled case-by-case basis 🤔

Reply:
I'll close this issue for now since we are not using it, but happy to add it later if this is something that people frequently need 😀